### PR TITLE
Add release toolkit dictionary (NR-112268)

### PR DESCRIPTION
**Pull request by gsanchezgavier**
Adds a common dictionary used by the release toolkit on automatic pre-releases.

### CODE REVIEW

File: rt-dictionary/.rt-dictionary.yml
Comment (gsanchezgavier) [inline rt-dictionary/.rt-dictionary.yml:1]: I will change default location and name of the file in the workflow that is used to run the rt

**Review by sigilioso — APPROVED** [review]
LGTM 👍 

Did you consider updating the README to mention this new file? (or the new configurable field)

**Comment by gsanchezgavier** [comment]
> Did you consider updating the README to mention this new file? (or the new configurable field)
Thanks for pointing it @sigilioso ,I totally forgot. 
I modified and added some notes in the readme. LMKWDYT

**Comment by sigilioso** [comment]
It looks great! Thanks for including the docs 🙂

**Pull request merged**